lib/ansible/modules/packaging/os/redhat_repositories.py

```diff
@@ -24,56 +24,58 @@
 module: redhat_repository
 short_description: Manage repositories with RHSM using the C(subscription-manager) command
 description:
-    - Manage repositories with the Red Hat Subscription Management entitlement platform using the C(subscription-manager) command
+  - Manage repositories with the Red Hat Subscription Management entitlement platform using the C(subscription-manager) command
```

Comment:
I knew why I had ts=4 sw=4 in my module. 4-spaces tabs are mandatory for the automated review to pass.
Furthermore, Python being unable to break line, why should I do it on documentation as well?